### PR TITLE
Add config files

### PR DESCRIPTION
**Pull request by FrancisBehnen**

### CODE REVIEW

**Comment by FrancisBehnen** [comment]
Just edit the config files for the admin project

File: polymer.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "entrypoint": "index.html",
+  "shell": "src/lancie-app.html",
+  "sourceGlobs": [
+   "images-optimized/**/*",
+   "scripts/*"
+  ],
+  "fragments": [
+    "src/lancie-home-page/lancie-home-page.html",
```
Comment (TimvdLippe) [inline polymer.json]: You can remove these for now

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+      .pipe(gulpif(/\.css$/, cssSlam()))
+      .pipe(gulpif(/\.html$/, cssSlam()))
+      .pipe(gulpif(/\.html$/, minify()))
+      .pipe(project.rejoin());
+}
+
+const root = path.resolve(process.cwd(), 'images');
+const optimizedImagesRoot = path.resolve(process.cwd(), 'images-optimized');
+const imageOptions = {
+  activities: '340x340',
```
Comment (TimvdLippe) [inline gulpfile.js]: This object should be empty as there are no images yet in `lancie-admin`

**Comment by FrancisBehnen** [comment]
Is de lancie-admin repo al gekoppeld aan het slack channel trouwens?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "private": true,
+  "devDependencies": {
+ },
+  "scripts": {
+    "build": "node ./node_modules/gulp/bin/gulp.js",
+    "ensure-images": "node ./node_modules/gulp/bin/gulp.js ensure-images-optimized",
+    "serve": "yarn run ensure-images && node ./node_modules/polyserve/bin/polyserve -p 5101 -P h2 --proxy-path api --proxy-target http://localhost:9000/api/",
+    "serve-build": "yarn run ensure-images && node ./node_modules/polyserve/bin/polyserve build/bundled/ -p 5101 -P h2 --proxy-path api --proxy-target http://localhost:9000/api/"
```
Comment (TimvdLippe) [inline package.json]: We should also fix this in `lancie-frontend`, but this line is redundant. We can (apparently) already do `yarn run serve build/bundled`

**Review by TimvdLippe — APPROVED** [review]
Small comments, other than that LGTM

File: package.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  "private": true,
+  "devDependencies": {
+ },
+  "scripts": {
+    "build": "node ./node_modules/gulp/bin/gulp.js",
+    "ensure-images": "node ./node_modules/gulp/bin/gulp.js ensure-images-optimized",
+    "serve": "yarn run ensure-images && node ./node_modules/polyserve/bin/polyserve -p 5101 -P h2 --proxy-path api --proxy-target http://localhost:9000/api/"
+  },
+  "engines": {
+    "node": ">=5.0.0"
```
Comment (martijnjanssen) [inline package.json]: @TimvdLippe we are running on travis node version 6.0.0, should we update this, too?

File: polymer.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{
+  "entrypoint": "index.html",
+  "shell": "src/lancie-admin-app.html",
+  "sourceGlobs": [
+   "images-optimized/**/*",
+   "scripts/*"
```
Comment (martijnjanssen) [inline polymer.json]: I don't think we need this folder in this repo. I vote for removal of this one.

File: package.json (hunk 4 above)
Comment (TimvdLippe) [reply package.json]: Yes we can

File: sw-precache-config.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+ "staticFileGlobs": [
+   "/index.html",
+   "/manifest.json",
+   "/images-optimized/**",
+   "/src/**",
+   "/bower_components/webcomponentsjs/webcomponents-lite.min.js"
+ ],
+ "navigateFallback": "/index.html",
+ "runtimeCaching": [{
+   "urlPattern": "/https:\\/\\/fonts\\.googleapis\\.com\\//",
```
Comment (martijnjanssen) [inline sw-precache-config.json]: Go ahead and remove the font. lets try to go for minimal size in this repo and use system fonts only. :package:

File: bower.json [hunk 7]
```diff
@@ -2,13 +2,30 @@
...
+    "iron-elements": "PolymerElements/iron-elements#^1.0.10",
+    "paper-elements": "PolymerElements/paper-elements#^1.0.7",
+    "gold-elements": "PolymerElements/gold-elements#^1.0.1",
+    "neon-elements": "PolymerElements/neon-elements#^1.0.0",
+    "google-map": "GoogleWebComponents/google-map#~1.1.14",
+    "polymer": "polymer/polymer#^1.7.0",
+    "wisvch-footer": "wisvch/wisvch-footer#^1.0.1",
+    "app-layout": "PolymerElements/app-layout#^0.10.4",
+    "app-route": "polymerelements/app-route#^0.9.2",
+    "app-storage": "PolymerElements/app-storage#^0.9.8",
```
Comment (martijnjanssen) [inline bower.json]: The app storage can be removed for now.

File: bower.json [hunk 8]
```diff
@@ -2,13 +2,30 @@
   "name": "lancie-admin",
   "description": "Admin panel for the LANcie API",
   "main": "index.html",
+  "private": true,
```
Comment (martijnjanssen) [inline bower.json]: I'm not sure we need this flag

File: gulpfile.js [hunk 9]
```diff
@@ -100,6 +101,10 @@ function dependencies() {
 const root = path.resolve(process.cwd(), 'images');
 const optimizedImagesRoot = path.resolve(process.cwd(), 'images-optimized');
 const imageOptions = {
+  activities: '340x340',
+  logos: '250,scale-down',
+  unofficial: '340x340',
+  slider: '2000x500,scale-down'
```
Comment (martijnjanssen) [inline gulpfile.js]: We won't be needing the slider, unofficial and activities image options.

File: package.json [hunk 10]
```diff
@@ -1,34 +1,31 @@
...
-  },
-  "name": "lancie-admin",
-  "version": "0.0.1",
-  "description": "Admin panel for the LANcie API",
-  "main": "index.js",
-  "repository": "https://github.com/AreaFiftyLAN/lancie-admin",
-  "author": "LANcie",
-  "license": "BSD-3-Clause",
-  "dependencies": {}
+    "node": ">=5.0.0"
```
Comment (martijnjanssen) [inline package.json]: Why not node 6.0.0?

File: polymer.json [hunk 11]
```diff
@@ -2,9 +2,29 @@
   "entrypoint": "index.html",
   "shell": "src/lancie-admin-app.html",
   "sourceGlobs": [
-   "images-optimized/**/*"
+   "images-optimized/**/*",
+   "scripts/*"
```
Comment (martijnjanssen) [inline polymer.json]: We do not have scripts yet, and may never have them. In the frontend we use them for data. I'm pretty sure we don't need that in this repo.

File: polymer.json [hunk 12]
```diff
@@ -2,9 +2,29 @@
...
+    "src/lancie-login/lancie-check-email.html",
+    "src/lancie-contact/lancie-contact.html",
+    "src/lancie-contact/lancie-contact-success.html",
+    "src/lancie-order-check.html",
+    "src/lancie-404.html",
+    "src/lancie-construction.html",
+    "src/lancie-password/lancie-password-reset.html",
+    "src/lancie-password/lancie-password-reset-success.html",
+    "src/lancie-password/lancie-request-password-reset.html",
+    "src/lancie-password/lancie-request-password-reset-success.html"
```
Comment (martijnjanssen) [inline polymer.json]: All these pages can be removed as of now, we do not require these for lazy-loading.

File: sw-precache-config.json [hunk 13]
```diff
@@ -7,5 +7,8 @@
    "/bower_components/webcomponentsjs/webcomponents-lite.min.js"
  ],
  "navigateFallback": "/index.html",
- "runtimeCaching": []
+ "runtimeCaching": [{
+   "urlPattern": "/https:\\/\\/fonts\\.googleapis\\.com\\//",
```
Comment (martijnjanssen) [inline sw-precache-config.json]: No fonts are needed right now.

**Review by martijnjanssen — CHANGES_REQUESTED** [review]
Looks good, but lets remove some stuff. :)

File: bower.json [hunk 14]
```diff
@@ -3,12 +3,27 @@
   "description": "Admin panel for the LANcie API",
   "main": "index.html",
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.4.0"
+    "iron-elements": "PolymerElements/iron-elements#^1.0.10",
+    "paper-elements": "PolymerElements/paper-elements#^1.0.7",
+    "gold-elements": "PolymerElements/gold-elements#^1.0.1",
+    "neon-elements": "PolymerElements/neon-elements#^1.0.0",
+    "google-map": "GoogleWebComponents/google-map#~1.1.14",
```
Comment (martijnjanssen) [inline bower.json]: Google maps? :)

File: .travis.yml [hunk 15]
```diff
@@ -0,0 +1,13 @@
+language: node_js
+node_js:
+  - '6'
+before_script:
+  - 'yarn global add bower'
+  - 'bower install'
+script: "yarn run build"
```
Comment (martijnjanssen) [inline .travis.yml]: Single quotes?

**Review by martijnjanssen — CHANGES_REQUESTED** [review]
I think we have almost fixed everything now.

File: gulp-tasks/project.js [hunk 16]
```diff
@@ -0,0 +1,138 @@
...
+    bundled: true
+  });
+}
+
+// Returns a Promise to generate a service worker for unbundled output
+function writeUnbundledServiceWorker() {
+  return polymer.addServiceWorker({
+    project: project,
+    buildRoot: unbundledPath,
+    swConfig: global.config.swPrecacheConfig,
```
Comment (TimvdLippe) [inline gulp-tasks/project.js]: Please also incorporate the changes of https://github.com/AreaFiftyLAN/lancie-frontend/pull/393

File: package.json (hunk 3 above)
Comment (TimvdLippe) [reply package.json]: Yup removed, per https://github.com/AreaFiftyLAN/lancie-frontend/pull/393

File: polymer.json [hunk 17]
```diff
@@ -0,0 +1,15 @@
+{
+  "entrypoint": "index.html",
+  "shell": "src/lancie-admin-app.html",
+  "sourceGlobs": [
```
Comment (TimvdLippe) [inline polymer.json]: `sources` per https://github.com/AreaFiftyLAN/lancie-frontend/pull/393

File: polymer.json [hunk 18]
```diff
@@ -0,0 +1,15 @@
...
+   "images-optimized/**/*"
+  ],
+  "fragments": [
+  ],
+  "includeDependencies": [
```
Comment (TimvdLippe) [inline polymer.json]: `extraDependencies` per https://github.com/AreaFiftyLAN/lancie-frontend/pull/393

**Review by TimvdLippe — CHANGES_REQUESTED** [review]

**Review by martijnjanssen — APPROVED** [review]

**Review by TimvdLippe — APPROVED** [review]

**Pull request merged**